src/components/pages/Login/LoginContainer.js

```diff
@@ -56,65 +33,25 @@ const LoginContainer = () => {
       },
     });
 
-    if (login) {
-      widget.renderEl(
-        { el: '#sign-in-widget' },
-        () => {
-          /**
-           * In this flow, the success handler will not be called because we redirect
-           * to the Okta org for the authentication workflow.
-           */
-        },
-        err => {
-          throw err;
-        }
-      );
-    } else {
-      widget.remove();
-    }
-  }, [login]);
+    widget.renderEl(
+      { el: '#sign-in-widget' },
+      () => {
+        /**
+         * In this flow, the success handler will not be called because we redirect
+         * to the Okta org for the authentication workflow.
+         */
+      },
+      err => {
+        throw err;
+      }
+    );
+  }, []);
 
   return (
-    <LoginDiv>
-      <Header>Welcome to Human Rights First©</Header>
-
-      <InfoDiv>
-        {/* Logo, application name, 3 icons with single line description */}
-        <LeftSide>
-          <img src={require('./hrf-logo.png')} alt="An Image" />
-          <h2>
-            Refugee <br />
-            Asylum <br />
-            Case <br />
-            Database
-          </h2>
-        </LeftSide>
-        <RightSide>
-          <div>
-            <TableChartOutlinedIcon style={{ fontSize: '6rem' }} />
-            <p>Perform custom queries and analyze data</p>
-          </div>
-          <div>
-            <p>Upload &amp; download cases in PDF or CSV</p>
-            <CloudDoneIcon style={{ fontSize: '6rem' }} />
-          </div>
-          <div>
-            <DescriptionOutlinedIcon style={{ fontSize: '6rem' }} />
-            <p>Access detailed reports on Judges and Cases</p>
-          </div>
-        </RightSide>
-      </InfoDiv>
-
-      <LoginButtonDiv>
-        {!login && (
-          <SecureLogin onClick={toggleLogin}>Secure Login</SecureLogin>
-        )}
-      </LoginButtonDiv>
-
-      {login && <div id="sign-in-widget" />}
-      {login && <CloseLogin onClick={toggleLogin}>X</CloseLogin>}
-      {login && <OverlayDiv></OverlayDiv>}
-    </LoginDiv>
+    <StyledLogin>
+      <div class="background-image" aria-label="cosmetic background image" />
+      <div id="sign-in-widget" aria-label="login form" />
+    </StyledLogin>
```

Comment:
Much cleaner, easier for the user to find the login form & sign in to their account, style closer to that of the human rights asylum website, not cluttered up by words and logos